examples/backend-adapters/server/wrangler.toml

```diff
@@ -0,0 +1,4 @@
+name = "uploadthing-cf-worker"
+main = "src/cf-worker.ts"
+compatibility_date = "2023-11-21"
+node_compat = true
```

Comment:
the only reason this is currently needed is that our logger uses `util.inspect`. Otherwise it works wihtout node_compat on. Unsure how to make it just "ignore" that import... we have fallback logic to use `JSON.stringify` but cf just throws when it sees the import (maybe that makes sense?)

idk if you can make a conditional export module for util that will make it accept it and then we just put a dummy function that throws not-implemented and we'll fallback to JSON.stringify?